Pipfile

```diff
@@ -0,0 +1,16 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+irc3 = "*"
+redis = "*"
+lxml = "*"
+aiohttp = "*"
+aiocron = "*"
+
```

Comment:
You forgot to add the **docopt**, **venusian** and **watchdog** libraries. Are they no longer needed?

Also, I was wondering if any of these libraries is a dependency of another.

For example, if **lxml** is required by **aiohttp** but it is not being directly accessed by our code we could omit and let the Pipenv manage that dependecy whitout put it on the Pipfile.

Reply:
**docopt** and **venusian** are being handled as dependency through Pipfile.lock, so they don't really need to be on Pipfile itself. In the other hand **watchdog** is needed.

About **lxml**, it's actually used in our code on plugins/behaviors.py#L131 and with that I think it's important to be a "hard" requirement for us, even being a dependency for **aiohttp**. HOWEVER, we might have a problem with this package being installed through pip: besides it requires either gcc and python3-devel to be installed (and other headers) it also took more than 30minutes to be compiled on my cubieboard2... I removed it and used the package available on linux distro repo since it's a pre-compiled one. I know it's platform dependent and in normal machines it wouldn't take that long, but what do you guys think about require a distro package for this one? Is there a way to pip explicitly require a pre-installed package?

Reply:
For some reason I haven't saw your previous comment when I started to write my review. 

About the platform provided packages: https://pipenv.readthedocs.io/en/latest/advanced/#working-with-platform-provided-python-components